integration-tests/src/test/java/tachyon/security/WorkerClientAuthenticationIntegrationTest.java

```diff
@@ -48,89 +49,56 @@
 
   @Before
   public void before() throws Exception {
-    mLocalTachyonCluster = new LocalTachyonCluster(1000, 1000, Constants.GB);
     mExecutorService = Executors.newFixedThreadPool(2);
     clearLoginUser();
   }
 
   @After
   public void after() throws Exception {
-    // stop cluster
-    mLocalTachyonCluster.stop();
-
     System.clearProperty(Constants.SECURITY_LOGIN_USERNAME);
   }
 
   @Test
+  @LocalTachyonClusterResource.Config(
+      tachyonConfParams = {Constants.SECURITY_AUTHENTICATION_TYPE, "NOSASL"})
   public void noAuthenticationOpenCloseTest() throws Exception {
-    // no authentication type configure
-    TachyonConf testConf = mLocalTachyonCluster.newTestConf();
-    testConf.set(Constants.SECURITY_AUTHENTICATION_TYPE, AuthType.NOSASL.getAuthName());
-
-    // start cluster
-    mLocalTachyonCluster.start(testConf);
-
     authenticationOperationTest();
-
-    // stop cluster
-    mLocalTachyonCluster.stop();
   }
 
   @Test
+  @LocalTachyonClusterResource.Config(
+      tachyonConfParams = {Constants.SECURITY_AUTHENTICATION_TYPE, "SIMPLE"})
   public void simpleAuthenticationOpenCloseTest() throws Exception {
-    // simple authentication type configure
-    TachyonConf testConf = mLocalTachyonCluster.newTestConf();
-    testConf.set(Constants.SECURITY_AUTHENTICATION_TYPE, AuthType.SIMPLE.getAuthName());
-
-    // start cluster
-    mLocalTachyonCluster.start(testConf);
-
     authenticationOperationTest();
-
-    // stop cluster
-    mLocalTachyonCluster.stop();
   }
 
   @Test
+  @LocalTachyonClusterResource.Config(
+      tachyonConfParams = {Constants.SECURITY_AUTHENTICATION_TYPE, "CUSTOM",
+          Constants.SECURITY_AUTHENTICATION_CUSTOM_PROVIDER,
+          NameMatchAuthenticationProvider.FULL_CLASS_NAME},
```

Comment:
You can't run any actual java code in these annotations, so the best I could do is have a constant.